ConnorAndersonLarson/PlanetParty
# Refactor/planetarium

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?


### Are there any known bugs?


### Summary of the changes
Planet data on main page now changes based on sort parameters, with descriptor (e.g. sorting by moons will return the planets with data displayed as 'x moons' based on how many moons each planet has).

When a planet is hovered over the text changes color to either a yellow or blue, dependent on the color of the planet (the exception being mercury, which has no color, currently the text remains gray)

### What ticket(s) do the changes resolve?
Close #31 

Please Link Issues to this PR as necessary.
#31 

### Notes for the Reviewer
This has only been tested on desktop view. May need to tweak display for touch based devices.

### How to Test Changes?
Enter this branch, run `npm start` 
Use sort function to change planet info/rankings, observe data below planet name
Hover over planets to view text change color; this is on a one second transition.

### Next Iterations
N/A
